I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 23, in script
    $ miniGame1 = HakingGame(100)
  File "game/script.rpy", line 23, in <module>
    $ miniGame1 = HakingGame(100)
  File "game/mini-game2.rpy", line 11, in __init__
    self.d2 = self.d.copy()
AttributeError: 'str' object has no attribute 'copy'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 23, in script
    $ miniGame1 = HakingGame(100)
  File "G:\Program Files (x86)\renpy-8.1.2-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "G:\Program Files (x86)\renpy-8.1.2-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 23, in <module>
    $ miniGame1 = HakingGame(100)
  File "game/mini-game2.rpy", line 11, in __init__
    self.d2 = self.d.copy()
AttributeError: 'str' object has no attribute 'copy'

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.2.23090503
ProjectOPD 0.3-EARLY-BETA
Thu Dec 14 17:07:19 2023
